src/reference/00-Getting-Started/01-Setup/a.md

```diff
@@ -9,28 +9,24 @@ out: Installing-sbt-on-Mac.html
 Installing sbt on Mac
 ---------------------
 
+### Installing from a universal package
+
+Download [ZIP][ZIP] or [TGZ][TGZ] package, and expand it.
+
 ### Installing from a third-party package
 
 > **Note:** Third-party packages may not provide the latest version. Please make
 > sure to report any issues with these packages to the relevant
 > maintainers.
 
-#### [Macports](http://macports.org/)
-
-```
-\$ port install sbt
-```
-
 #### [Homebrew](http://mxcl.github.com/homebrew/)
 
 ```
 \$ brew install sbt
 ```
 
-### Installing from a universal package
-
-Download [ZIP][ZIP] or [TGZ][TGZ] package, and expand it.
-
-### Installing manually
+#### [Macports](http://macports.org/)
 
-See instruction to install manually.
+```
+\$ port install sbt
+```
```

Comment:
@milessabin In my experience, sbt-extras is considerably more reliable at dealing with a number of cases than the vanilla launcher, including sbt version.  I've also seen occasional really bizarre dependency symptoms in someone else's build that were fixed by swapping from the vanilla launcher to sbt-extras.  It's still the standard, in my book.